app/src/main/java/org/akvo/flow/presentation/settings/SendDeviceInfoDialog.kt

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (C) 2018-2019 Stichting Akvo (Akvo Foundation)
```

Comment:
Is it a new file?

Reply:
no, I had copied from another file and forgot to update the copyright